scripts/doc.Makefile

```diff
@@ -0,0 +1,50 @@
+SOURCES=$(shell find content/ -name '*.md')
```

Comment:
As noted earlier, I'd personally attempt writing this in Nix to keep things more deterministic and tooling footprint as little as possible. Makefiles are notorious for being complex; but this is a minor comment.

Reply:
The nice thing with a `Makefile` is that if you change a single `.md` source file, a single `.html` will be re-generated when invoking `make`. If we use Nix, I guess everything will be built again when invoking `nix-build -A documentation` (assuming we would have a  `documentation` attribute. Or do you see another way ?